src/Microsoft.DotNet.Build.Tasks/PackageFiles/msbuild.cmd

```diff
@@ -0,0 +1,8 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+echo msbuild.exe %*
```

Comment:
At some point I want to make this configurable so we could chose to use the .NET Core msbuild as well.